tpe/plugin/src/SimulationFeatures_TEST.cc

```diff
@@ -230,10 +230,6 @@ TEST_P(SimulationFeatures_TEST, FreeGroup)
     auto frameData = model->GetLink(0)->FrameDataRelativeToWorld();
     EXPECT_EQ(ignition::math::Pose3d(0, 0, 2, 0, 0, 0),
               ignition::math::eigen3::convert(frameData.pose));
-    EXPECT_EQ(ignition::math::Vector3d(0.5, 0, 0.1),
-              ignition::math::eigen3::convert(frameData.linearVelocity));
-    EXPECT_EQ(ignition::math::Vector3d(0.1, 0.2, 0),
-              ignition::math::eigen3::convert(frameData.angularVelocity));
```

Comment:
do we need to do something to fix the linear and angular vel?

Reply:
My understanding is that linear and angular velocity are tied to `model` instead of `link`, and this test case started checking the velocity of `link` after I corrected the implementation of `framedata`, which is why I removed it. Before it worked because when calling `link->FrameDataRelativeToWorld()`, it was getting the velocity of the link's parent model. 

If we do want to add velocity for links, then that would be a different feature which allows links to move by itself.

Reply:
oh I see, that makes sense. Just checking, if we assume the link does not move within a model, should it inherit the parent model's velocities in world frame?

In any case, I'm not sure if there's a use case for querying a link's world velocities. We can address this when the need comes.

Reply:
If the link doesn't move within model, its `WorldPose` is then updated as model's `WorldPose` gets updated (since its offset will remain the same), so we probably don't have to keep track of link velocities in world frame. 

However, if we want the link to move within model, then we'll have to compute the link velocities with the offset. And yes, I agree that we can add this if the need comes.